clients/python/docs/CommitsApi.md

```diff
@@ -67,6 +67,7 @@ with lakefs_client.ApiClient(configuration) as api_client:
         metadata={
             "key": "key_example",
         },
+        date=1,
```

Comment:
Maybe give an example of a real date, 1640988000 (1/1/22)

Reply:
This code is generated, tried adding an example in the sagger file, didn't seem to affect the python docs.
leaving as is
Thanks